docs/available-runtimes.rst

```diff
@@ -0,0 +1,76 @@
+Available Runtimes
+==================
+
+This page provides information about available Flatpak runtimes. It is primarily intended as information for application developers and distributors.
+
+There are currently three main runtimes available: Freedesktop, GNOME and KDE. These are all hosted on `Flathub <https://flathub.org/>`_.
+
+Freedesktop
+-----------
+
+The Freedesktop runtime is the standard runtime that can be used for any application and contains a set of essential libraries and services, including D-Bus, GLib, PulseAudio, X11 and Wayland.
+
+Available Freedesktop runtimes:
+
+===============================  =================================
+ID                               Description
+===============================  =================================
+org.freedesktop.Platform         Runtime
+org.freedesktop.Platform.Locale  Runtime translations (extension)
+org.freedesktop.Sdk              SDK
+org.freedesktop.Sdk.Debug        SDK debug information (extension)
+org.freedesktop.Sdk.Locale       SDK translations (extension)
+org.freedesktop.Sdk.Docs         SDK documentation (extension)
+===============================  =================================
+
+GNOME
+-----
+
+The GNOME runtime is appropriate for any application that uses the GNOME platform. It is based on the Freedesktop runtime and adds the GNOME platform, including:
+
+* AT-SPI
+* ATK (Accessibility Toolkit)
+* Clutter
+* dconf
+* Gjs
+* GObject Introspection
+* GStreamer
+* GTK+
```

Comment:
`gtk3` is actually in the freedesktop runtime and most if its deps are not important like atk/dconf/at-spi.

Reply:
OK, I've removed GTK, AT-SPI, ATK and GTK+ from the list.